OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -110,6 +110,8 @@ public class UpdatePath
 				new RemoveResourceExplodeModifier(),
 				new DefineLevelUpImageDefault(),
 				new RemovedAutoCarryallCircleTurnSpeed(),
+				new ReplacedWithChargeAnimation(),
+				new RenameWithResources(),
```

Comment:
This seems to be missing `ReplacedWithSiloAnimation`

Reply:
Fixed.